cmd/kubeadm/app/apis/kubeadm/defaults.go

```diff
@@ -20,4 +20,6 @@ const (
 	DefaultServiceDNSDomain  = "cluster.local"
 	DefaultServicesSubnet    = "10.12.0.0/12"
 	DefaultKubernetesVersion = "v1.4.1"
+	DefaultAPIBindPort       = 6443
```

Comment:
We had it initially at 443, but it seemed more Kubernetes-default-y to use a n443 port because that's over 1000 like everything else and a lot of users complained they already had something on 443.